its/plugin/tests/src/test/java/com/sonar/it/java/suite/JaCoCoControllerTest.java

```diff
@@ -52,8 +52,13 @@ public void setUp() throws Exception {
     Optional<String> javaVersion = new Gson().fromJson(json, Plugins.class).getPlugins()
       .stream()
       .filter(plugin -> plugin.getKey().equals("java"))
-      .map(Plugins.Plugin::getVersion)
-      .findFirst();
+      .map(p -> {
+        String f = p.filename;
+        if(f.contains("SNAPSHOT")) {
+          return p.version;
+        }
+        return f.substring(f.lastIndexOf('-') + 1, f.lastIndexOf('.'));
```

Comment:
I would add a comment to describe in which context the artifact won't be tagged with "SNAPSHOT".